zp-relayer/endpoints.ts

```diff
@@ -205,14 +206,26 @@ function relayerInfo(req: Request, res: Response) {
   })
 }
 
-function getFee(req: Request, res: Response) {
+// NOTE: Will be deprecated
+function getStaticFee(req: Request, res: Response) {
```

Comment:
I think static fees still might be useful for some of our future deployments, e.g. having a static 0.01$ fee for GC deployments might be a good idea.

Can we support all 3 schemes: static, dynamic, dynamic-optimism under the same `/fee` endpoint? We can add a fee manager type field in the JSON response if needed.

Reply:
Good point about GC. I think we can have just one endpoint configurable via some env parameter.

Reply:
I think so

Reply:
Done